docs/components/datetimepicker.md

```diff
@@ -0,0 +1,135 @@
+# DateTimePicker
+
+<div v-if="hasMounted">
+
+**KDateTimePicker** - A date and time selection tool, displayed inside a popover.
+
+## Examples
+
+Display both a calendar and relative time frames, passing in a preset week-long range.
+
+<KCard>
+  <template v-slot:body>
+    <div class="mt-3">Emitted value: <pre>{{ currentValue1 }}</pre></div>
+    <KDateTimePicker
+      @changed="newVal => currentValue1 = newVal"
+      defaultMessage="Please select a date or time range"
+      :timePeriods="exampleTimeFrames"
+      :defaultValue="defaultRangeWeek"
+      v-model="currentValue1"
```

Comment:
`v-model` is a sytax sugar for `value` prop and `input` event. Since we have none inside `KDateTimePicker`, I think adding `v-model` here has no effect.

Reply:
`currentValue1` and `currentValue2` are using `v-model` for demo purposes in that specific markdown files, so I can reflect what values would get emitted to the parent in the `<pre>` tag above the component.

Reply:
```
@changed="newVal => currentValue1 = newVal"
```
This line already guarantees the emitted value is passed to `currentValue1`.

Using `v-model` on `KDateTimePicker` will only work if `KDateTimePicker` itself has a `value` prop and emits `input` events.

You can try removing `v-model="currentValue1"` and the example should still work as expected.